test/builder-api/src/origins_deletion.js

```diff
@@ -0,0 +1,382 @@
+const expect = require('chai').expect;
```

Comment:
Is there any particular reason this didnt get inlined into the pre-existing origin deletion tests in origins.js ?

Reply:
Honestly due to formatting differences. I was able to collapse everything into `origin.js` though in https://github.com/habitat-sh/builder/pull/1298/commits/c730a638e289f05fa77022bc3012125b1917a039